src/color-wheel-picker/index.js

```diff
@@ -158,21 +158,30 @@ export default class ColorWheel extends Component {
   };
 
   forceUpdate = color => {
-    const {h, s, v} = colorsys.hex2Hsv(color);
-    const {left, top} = this.calcCartesian(h, s / 100);
-    this.setState({currentColor: color, inputText: color});
-    this.props.onColorChange(color);
-    this.state.pan.setValue({
-      x: left - this.props.thumbSize / 2,
-      y: top - this.props.thumbSize / 2,
-    });
+    try {
+      const {h, s, v} = colorsys.hex2Hsv(color);
+      const {left, top} = this.calcCartesian(h, s / 100);
+      this.setState({currentColor: color, inputText: color});
+      this.props.onColorChange(color);
+      this.state.pan.setValue({
+        x: left - this.props.thumbSize / 2,
+        y: top - this.props.thumbSize / 2,
+      });
+    }
+    catch (err) {
+      console.log(err);
+    }
   };
 
   changeInput = text => {
-    if (text[0] === '#' && text.length === 7) {
-      this.forceUpdate(text);
+    try {
+      if (text[0] === '#' && text.length === 7) {
+        this.forceUpdate(text);
+      }
+      this.setState({inputText: text});
```

Comment:
Maybe it is better to put this inside an `else`?
Because in the `if` condition above, inside the `forceUpdate` method, we're already doing `this.setState({currentColor: color, inputText: color});`
Also, it would be wise to change the variable name to `color` instead of `text`. since it is a color.

Reply:
Good knock, Thanks!